isotovideo

```diff
@@ -246,6 +250,11 @@ $bmwqemu::vars{TEST_GIT_HASH} = $test_git_hash;
 # is not supposed to talk to the backend directly
 ($cpid, $cfd) = commands::start_server($bmwqemu::vars{QEMUPORT} + 1);
 
+# rsync before we read main.pm - it might trigger opencv and we need
+# to move this as late as possible due to the evil udev thread hooks
+# opencv adds
+$shared_cache = sync_needles_to_cache;
```

Comment:
i'd change this to
` sync_needles_to_cache if $bmwqemu::vars{CACHEDIRECTORY}; `
just to remove that `if` inside the   `sync_needles_to_cache` and that extra variable. But other than that looks good to me